pages/api/getlogs.tsx

```diff
@@ -0,0 +1,15 @@
+import * as fs from 'fs';
+import getConfig from 'next/config';
+
+const { serverRuntimeConfig } = getConfig();
+
+const LOG_PATH = serverRuntimeConfig.REACT_APP_LOG_FILEPATH;
+
+export default async function handler(_req: any, res: any) {
+  if (!LOG_PATH) {
```

Comment:
This will expose the logs publically, are we sure we want that? And not just directly check the server by hand whenever needed?

Perhaps a good solution might be to use:
```js
if (process.env.NODE_ENV !== 'development') {
    res.status(404) // or whatever
    return;
}
```
and expose them only in development mode

Reply:
I would like Michael and other kwarcies access this easily. I feel like SSHing and getting the file is going to be a barrier that will slow down folks. Added a simple passcode scheme to make sure not everyone can access the log file.

Reply:
I like the idea, but this should really be HTTP Basic instead of a custom scheme.

Reply:
To be solved later, see #37.